developer_docs/README.md

```diff
@@ -24,16 +24,16 @@ We recognize all types of contributions. This project follows the [all-contribut
 
 Aside from the code itself, you may also need to supply some combination of the following.
 
-- [inline documentation](https://github.com/processing/p5.js/blob/master/developer_docs/inline_documentation.md) in the form of code comments, which explain the code both to other developers and to users. Many of these comments must conform to [JSDoc](https://usejsdoc.org) syntax and will be published on the p5.js website as part of the [online reference manual](https://p5js.org/reference/)
-- [unit tests](https://github.com/processing/p5.js/tree/master/developer_docs#unit-tests), small pieces of code which are separate from the library and are used to verify its behavior
-- [benchmarks](https://github.com/processing/p5.js/blob/master/developer_docs/benchmarking_p5.md) to test performance
+- [inline documentation](./inline_documentation.md) in the form of code comments, which explain the code both to other developers and to users. Many of these comments must conform to [JSDoc](https://usejsdoc.org) syntax and will be published on the p5.js website as part of the [online reference manual](https://p5js.org/reference/)
+- [unit tests](./unit_testing.md), small pieces of code which are separate from the library and are used to verify its behavior
+- [benchmarks](./benchmarking_p5.md) to test performance
 
 ## Examples
 
-The p5.js site includes [integrated examples](http://p5js.org/examples/). You can [add more](https://github.com/processing/p5.js-website/blob/master/contributor_docs/Adding_examples.md), and there is an issue which lists some [desired examples](https://github.com/processing/p5.js/issues/1954).
+The p5.js site includes [integrated examples](http://p5js.org/examples/). You can [add more](https://github.com/processing/p5.js-website/wiki/Adding-examples), and there is an issue which lists some [desired examples](https://github.com/processing/p5.js/issues/1954).
```

Comment:
this was actually moved to here: https://github.com/processing/p5.js-website/blob/master/contributor_docs/Adding_examples.md

Reply:
Thank you for catching that!